app/controllers/games_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Comment necessary for Rubocop
+class GamesController < ApplicationController
+  before_action :set_game, only: %i[show update destroy]
+
+  # GET /games
+  def index
+    @games = Game.all
+
+    render json: @games
+  end
+
+  # GET /games/1
+  def show
+    render json: render_game_with_state_and_player(@game)
+  end
+
+  # POST /games
+  def create
+    Game.delete_all
+    new_game = TicTacToeLogic::GameBoardData.new
+    @game = Game.new({ "board": new_game })
+
+    if @game.save
+      render json: render_game_with_state_and_player(@game)
+    else
+      render json: @game.errors, status: :unprocessable_entity
+    end
+  end
+
+  # POST '/submit_move/:move'
+  def submit_move
+    @game = Game.find(1)
```

Comment:
Are we always using the same entry in the database??  How will multiple people be able to play different games if this is always grabbing the first entry?

Reply:
will Fix so users can play different games